<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="font">
        <font>
         <family>Roboto Mono</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1031</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Fira Code</family>
              <pointsize>24</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Simple Version Control System</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>190</y>
           <width>681</width>
           <height>101</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="1,4,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <family>Roboto Mono</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Fira Code</family>
              <pointsize>14</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Select a repository</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="browseBtn1">
            <property name="text">
             <string>Choose a folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="enterToRepo">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>320</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Enter</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QPushButton" name="addBtn">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>350</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
        <widget class="QPushButton" name="commitBtn">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>350</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Commit</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>240</y>
           <width>451</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Fira Code</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Select Files to add:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>300</y>
           <width>331</width>
           <height>28</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="browseBtn2">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>300</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="statusTextbox">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>150</y>
           <width>481</width>
           <height>411</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>1011</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Fira Code</family>
              <pointsize>16</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Repository :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="repoInitMessage">
            <property name="font">
             <font>
              <family>Roboto Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="changeRepoBtn">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>540</y>
           <width>181</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Change repository</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="commitSuccessLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>430</y>
           <width>461</width>
           <height>91</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimple_Version_Control_System">
    <property name="title">
     <string>Simple Version Control System</string>
    </property>
   </widget>
   <addaction name="menuSimple_Version_Control_System"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
